test/functional/currency/currency-formatter.js

```diff
@@ -19,7 +19,8 @@ define([
 
 var accounting = { style: "accounting" },
 	code = { style: "code" },
-	name = { style: "name" },
+    name = { style: "name" },
+	narrow = { symbolForm: "symbol-alt-narrow" },
```

Comment:
Re-thinking about it, `options.form` could cause confusion. With `options.symbolForm` we make it very explicit that this form is only available when `options.symbol` is `"symbol"`.